src/test/java/UserSmokeTests.java

```diff
@@ -8,54 +9,78 @@
 public class UserSmokeTests extends BaseTest {
 
     private static final String USER_ENDPOINT = BASE_URL + "/user";
-    private static final String USERNAME = "user_1";
+    private static final String NON_EXIST_USERNAME_ENDPOINT = USER_ENDPOINT + "/non_existing_username";
+    private static final String NON_EXIST_USER_ENDPOINT  = USER_ENDPOINT + "/user_not_existed";
+
+    @BeforeMethod
+    public void setUp() {
+        if (!userExists(DEFAULT_USERNAME)) {
+            createUser(DEFAULT_USERNAME, DEFAULT_FIRST_NAME, DEFAULT_LAST_NAME, DEFAULT_EMAIL, DEFAULT_PASSWORD, DEFAULT_PHONE, DEFAULT_USER_STATUS);
+        }
+    }
 
     @Test
     public void createUserTest() {
-        String requestBody = createUserJson(2, USERNAME, "firstName1", "lastName1", "email1@gmail.com", "qwerty", "12345678", 0);
-        Response response = sendRequest("POST", USER_ENDPOINT, requestBody);
-        verifyResponse(response, SUCCESS_STATUS_CODE, UNKNOWN_TYPE, "2");
+        String requestBody = createUserJson(
+                DEFAULT_USER_ID,
+                DEFAULT_USERNAME,
+                DEFAULT_FIRST_NAME,
+                DEFAULT_LAST_NAME,
+                DEFAULT_EMAIL,
+                DEFAULT_PASSWORD,
+                DEFAULT_PHONE,
+                DEFAULT_USER_STATUS
+        );
+        Response response = sendRequest(POST_METHOD, USER_ENDPOINT, requestBody);
+        verifyResponse(response, SUCCESS_STATUS_CODE, UNKNOWN_TYPE, Integer.toString(DEFAULT_USER_ID));
     }
 
     @Test
     public void getExistingUserTest() {
-        Response response = sendRequest("GET", USER_ENDPOINT + "/" + USERNAME, null);
+        Response response = sendRequest(GET_METHOD, USER_ENDPOINT + "/" + DEFAULT_USERNAME, null);
```

Comment:
You can simplify method signature to Response response = getUser(DEFAULT_USERNAME);

Here you don't need "GET", you don't need even endpoint since it could be moved to class with previous sendRequest